development/tools/cmd/gcrcleaner/README.md

```diff
@@ -0,0 +1,46 @@
+# GCR cleaner
+
+## Overview
+
+This command finds and removes old GCR images created by jobs in the Google Cloud Platform (GCP) project.
+
+There are three conditions used to find images for removal:
+- The repository name pattern is not on the ignored list.
+- The `creationTimestamp` value of the images, that is used to find addresses, exists at least for a preconfigured number of hours.
+
+GCR images that meet these conditions are subject to removal.
+
+## Usage
+
+For safety reasons, the dry-run mode is the default one.
+To run it, use:
+```bash
+env GOOGLE_APPLICATION_CREDENTIALS={path to service account file} go run main.go \
+    --repository={gcloud repository url}
```

Comment:
```suggestion
env GOOGLE_APPLICATION_CREDENTIALS={PATH_TO_SERVICE_ACCOUNT_FILE} go run main.go \
    --repository={GCLOUD_REPOSITORY_URL}
```
Please use capital letters for placeholders and separate the words with an underscore (`_`).